frontend/src/server/api/routers/ai/types/session/index.ts

```diff
@@ -460,6 +458,15 @@ interface AiSessionMessageToolOutputBody_Transfer {
 	token_address: string;
 }
 
+interface AiSessionMessageToolOutputBody_Swap {
+	amount: string;
+	chain_id: string; // Check `lib/wagmi/config` to see all supported chain IDs
```

Comment:
we could add typing.

first, declare typing as in https://wagmi.sh/react/typescript#declaration-merging. then:

```ts
import type { Register } from "wagmi"

//...
chain_id: Register["config"]["chains"][number]["id"]
//...
```

Reply:
Updated